.github/workflows/tests.yml

```diff
@@ -45,7 +48,7 @@ jobs:
           done
 
       - name: Check for CI-syntax-only label
```

Comment:
I wonder if we can combine all of the `Check for * label` steps into one and use `console.log("::set-output name= run-tests::true");`.

That way, we don't need to query the labels 4 separate times (and it makes the code shorter).

Reply:
good idea

Reply:
@rylan12 Done in d513bc0193